examples/baseball.py

```diff
@@ -0,0 +1,275 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import logging
+import math
+
+import numpy as np
+import pandas as pd
```

Comment:
can you add `pandas` to the `extra_requires` in `setup.py`

Reply:
I have added it; and its being tested in test-examples.